apps/web/components/Gallery/Gallery.vue

```diff
@@ -180,6 +180,22 @@ const registerThumbsWatch = (
 registerThumbsWatch(firstThumbReference, firstVisibleThumbnailIntersected);
 registerThumbsWatch(lastThumbReference, lastVisibleThumbnailIntersected);
 
+const getWidth = (image: ImagesData, imageUrl: string) => {
+  const imageWidth = productImageGetters.getImageWidth(image) || 600;
+  if (imageWidth && imageWidth > 0 && imageUrl.includes('/full/')) {
```

Comment:
`imageWidth > 0` => Redundant check since imageWidth is already truthy from `imageWidth &&`

Reply:
i agree, in the previous application had sens because imageWidth could have been undefined, null, 0, <=0.
If this assignment  of imageWidth is the prefered way (const imageWidth = productImageGetters.getImageWidth(image) || 600;)
Then both checks imageWidth && imageWidth > 0  can go.
But in product-card height and width recived throw props should stay like in lts.